docs/source/modin/supported/series_supported.rst

```diff
@@ -379,7 +379,8 @@ Methods
 | ``sort_index``              | P                               |                                  | ``N`` if given the ``key`` param,                  |
 |                             |                                 |                                  | or MultiIndex                                      |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``sort_values``             | P                               |                                  | ``N`` if given the ``key`` parameter               |
+| ``sort_values``             | P                               | ``key``                          | Snowpark pandas currently uses stable sort when    |
```

Comment:
What about DataFrame.sort_values?

Reply:
Should be the same for DataFrame.sort_values, did not think of that. I updated the documentation for that as well.

Reply:
Great.